### PR TITLE
Add ANTIC (Algebraic Number Theory in C)

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

File: recipes/antic/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+chmod +x configure
+
+export CFLAGS="$CFLAGS -funroll-loops -g -Wno-unknown-pragmas"
```
Comment (xhochy) [inline recipes/antic/build.sh:5]: What's the reason for the custom flags here?

Comment (saraedum) [reply recipes/antic/build.sh:5]: I copied them over from the FLINT recipe. But I guess we should try without first.
Thanks for the comment.

**Comment by saraedum** [comment]
@isuruf, since this is flint related, you might know what's behind this one?
```
    CC   ../build/nf/test/t-init_clear
dyld: Library not loaded: @rpath/libflint-14.dylib
  Referenced from: /usr/local/miniconda/conda-bld/antic_1592319301255/work/nf/../build/nf/test/t-init_clear
  Reason: image not found
```

**Comment by isuruf** [comment]
You need something like https://github.com/fredrik-johansson/arb/pull/300/files

**Comment by saraedum** [comment]
@conda-forge/help-c-cpp this one is ready for review :)

File: recipes/antic/build.sh (hunk 1 above)
Comment (isuruf) [reply recipes/antic/build.sh]: flint/antic's configure overrides these default flags if CFLAGS is set in an env variable. So, these default flags need to be added back in.

Comment (saraedum) [reply recipes/antic/build.sh]: Ah great. Let me revert that change then.

**Pull request merged**